File: android/app/src/main/java/com/hora/muthal/util/Categories.kt
package com.hora.muthal.util

/** Ports web lib/categories.ts / ios Categories.swift — keep in sync (SPEC §4). */
object Categories {
    val INCOME = listOf("Donation", "Offering", "Membership", "Grant", "Other")
    val EXPENSE = listOf("Salary", "Utilities", "Maintenance", "Supplies", "Event", "Other")

    fun forType(type: String): List<String> = when (type) {
        "income" -> INCOME
        "expense" -> EXPENSE
        else -> emptyList()
    }
}

File: ios/Muthal/Models/Categories.swift
import Foundation

/// Ports web lib/categories.ts / android Categories.kt — keep in sync (SPEC §4).
enum Categories {
    static let income = ["Donation", "Offering", "Membership", "Grant", "Other"]
    static let expense = ["Salary", "Utilities", "Maintenance", "Supplies", "Event", "Other"]

    static func forType(_ type: String) -> [String] {
        switch type {
        case "income": return income
        case "expense": return expense
        default: return []
        }
    }
}

let institutionTypes = ["Temple", "Church", "Library", "Other"]
